src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -21,54 +21,67 @@ export interface Props {
 
 interface FormattedPrices {
   priceFormatted: string
-  invertedPriceFormatted: string
+  inversePriceFormatted: string
+  inversePriceRaw: string
+  priceRaw: string
 }
 
-function formatPriceToPrecision(price: BigNumber): string {
-  return price.toFixed(PRICE_ESTIMATION_PRECISION)
+const LOW_PRICE_FLOOR = new BigNumber(0.0001)
```

Comment:
I think it's better to use string in the constructor, so there's never float point issues